Add company values to the about page

joeduffy opened the pull request:
We've been discussing this for a while now but understandably haven't been able to carve out the time due to lots of other priorities. My design skills are quite poor but I think it's an okay start.

![image](https://user-images.githubusercontent.com/3953235/138328870-3123b9e4-cf91-42d8-b22a-aa4116be1ff0.png)

I also slightly editorialized the values so they fit more cleanly on the page (we probably want to edit the internal doc down a little bit too, as it's a tad more verbose than it needs to be.)

susanev commented on themes/default/layouts/page/about.html:
```diff
@@ -1,25 +1,173 @@
+                            </div>
+                            <div class="description">
+                                <p class="text-sm">
+                                    We believe that working together is critical to our success
```
i think this should be rephrased into what we do instead of what we believe. something like "we work together..."

susanev commented on themes/default/layouts/page/about.html:
```diff
@@ -1,25 +1,173 @@
+                            </div>
+                            <div class="description">
+                                <p class="text-sm">
+                                    We are a global team and believe an inclusive and diverse
+                                    set of perspectives is essential to accomplishing our mission.
```
```suggestion
                                    We are a global and inclusive team with a diverse
                                    set of perspectives.
```

susanev commented on themes/default/layouts/page/about.html:
```diff
@@ -1,25 +1,173 @@
+                            <div class="description">
+                                <p class="text-sm">
+                                    We say yes more than we say no. Good today is better than
+                                    perfect tomorrow. When in doubt, experiment and fail fast.
```
this is maybe a separate convo, but fail fast has a negative connotation now with some folks so we may want to figure out different phrasing. i do think this work though just ending on experiment though.

susanev reviewed (approved):
i dont feel strongly about my suggested edits to the values, so feel free to only use what you agree with.

a few recommendations...

- [ ] we have been trying to standardize the nav items matching the page titles; this helps folks confirm they navigated to the place they intended, so id rec keeping the current title of about us
- [ ] the 4th piece of content in the second row looks really out of place, id recommend adding it as a subhead to the title instead
- [ ] id love to use icons that family better with the visual design on our website, no need to block though, i can follow up with improvements there in a separate pr

joeduffy commented:
@susanev I made some changes, including incorporating your suggestions to the descriptions, and making the title "About Us." I don't have a ton of time to work on other design elements but this feels like strictly a much better place to end up for now. Are you okay merging as-is and we can come back and do an overhaul when you and Casie are able to spend a bit more effort on overall design and flow? I know we can do better, but I would love to lead with our values on this page as soon as we can! :-)

susanev commented:
yes absolutely pls feel free to merge, thats why i approved when i left the comments!

joeduffy commented:
Oh, gotcha, sounds good! :+1: I always feel bad merging without addressing all feedback, but I'm now realizing almost two weeks later I might have bitten off more than I can chew. Appreciate the support merging this in 😄
